masterlist.yaml

```diff
@@ -693,6 +693,12 @@ globals:
     subs: [ '**Fores New Idles in Skyrim** or **Nemesis Unlimited Behavior Engine**' ]
     condition: 'file("Nemesis_Engine/Nemesis Unlimited Behavior Engine.exe") and file("tools/GenerateFNIS_for_Users/GenerateFNISforUsers.exe") and not file("scripts/FNISCreatureVersion.pex")'
 
+  # OAR and DAR compatibility
+  - <<: *useOnlyOneX
+    type: error
+    subs: [ '**Dynamic Animation Replacer** or **Open Animation Replacer**' ]
+    condition: 'file("SKSE/Plugins/DynamicAnimationReplacer.dll") and file("SKSE/Plugins/OpenAnimationReplacer.dll")'
+
```

Comment:
Please change the comment to `DAR and OAR compatibility`, so that it is in line with the `subs` content.